Orange/widgets/visualize/owscatterplot.py

```diff
@@ -380,10 +380,16 @@ def update_graph(self, attributes=None, reset_view=True, **_):
         self.graph.zoomStack = []
         if attributes and len(attributes) == 2:
             self.attr_x, self.attr_y = attributes
+        X = BasicStats(self.data[:, self.attr_x], self.attr_x)
+        Y = BasicStats(self.data[:, self.attr_y], self.attr_y)
+        if X.nans or Y.nans:
+            self.warning("Some of the attributes contain missing values") 
```

Comment:
Actually, here we know exactly which attributes are problematic, so explicit names are probably better than "Some attributes"
Or if att. names are not of interest (the user sees which ones are selected anyway), maybe something along the lines of:
"Some data instances are not shown due to missing values."